plugins/plugin.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<plugin id="com-optimobilecreate-plugins" version="1.0.0" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Plugin name / ID should be something like `optimove-cordova-plugin`, this will be the public name consumers use to install/reference the plugin.